package.json

```diff
@@ -35,22 +35,23 @@
         "@babel/core": "^7.21.3",
         "@babel/eslint-parser": "^7.21.3",
         "@babel/plugin-proposal-decorators": "^7.21.0",
-        "@nuxt/eslint-config": "^0.1.1",
         "@next/eslint-plugin-next": "^13.4.5",
+        "@nuxt/eslint-config": "^0.1.1",
         "@typescript-eslint/eslint-plugin": "^5.59.0",
         "@typescript-eslint/parser": "^5.59.0",
         "command-line-args": "^5.2.1",
         "command-line-usage": "^6.1.3",
+        "es-main": "^1.3.0",
         "eslint": "^8.38.0",
         "eslint-plugin-ember": "^11.4.8",
         "eslint-plugin-react": "^7.32.2",
         "eslint-plugin-vue": "^9.10.0",
         "expect.js": "^0.3.1",
-        "http-server": "^14.1.1",
         "mocha": "^10.2.0",
         "prettier": "^2.8.3",
         "selenium-webdriver": "^4.27.0",
         "sinon": "^17.0.1",
-        "typescript": "^5.0.4"
+        "typescript": "^5.0.4",
+        "local-web-server": "^5.4.0"
```

Comment:
I wonder if we should use `lws` with just the few middlewares we need (probably just log, cors, static, index?), instead of `local-web-server` with all the dependencies it comes with. What do you think?

Reply:
was easy enough, and limited it now to "lws", "lws-log", "lws-cors", "lws-static" and "lws-index"